modules/nf-core/repeatmodeler/repeatmodeler/main.nf

```diff
@@ -0,0 +1,54 @@
+process REPEATMODELER_REPEATMODELER {
+    tag "$meta.id"
+    label 'process_high'
```

Comment:
Does this need `process_high`? How much memory does it typically require?

Reply:
Thank you! That's a good point. REPEATMODELER's CPU utility is poor because of massive disk IO. `process_medium` should be more than adequate for it. I have updated the label.